data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessor.java

```diff
@@ -41,6 +41,7 @@ public class KeyValueProcessor extends AbstractProcessor<Record<Event>, Record<E
     private final Set<String> validTransformOptionSet = Set.of("", LOWERCASE_KEY, UPPERCASE_KEY, CAPITALIZE_KEY);
     private final String WHITESPACE_STRICT = "strict";
     private final String WHITESPACE_LENIENT = "lenient";
```

Comment:
Take it or leave it: we usually reserve capitalized variable name for static variables and use camel cases for instance variables. You could make those static